scripts/o3de/example_project_export_scripts/newspaper_delivery_project_export.py

```diff
@@ -0,0 +1,44 @@
+import sys
```

Comment:
This file, newspaper_delivery_project_export.py, should be in the newspaper delivery project folder and not in the global engine folder

Reply:
The author is aware of this issue - but it's here to make it easier to review the whole concept, it let this file be in this draft pull request for reference.

Reply:
Other feedback: For the deliverable, we can link to examples in the docs or elsewhere